scripts/lldb_pretty_printers/nx_core_array_pretty_printer.py

```diff
@@ -0,0 +1,83 @@
+import lldb
+
+def array_summary(valobj, internal_dict):
+    """
+    Custom summary function for nx::core::Array<T, Dimensions>.
+    Shows the elements of the private std::array<T, Dimensions> m_Array.
+    """
+
+    # 1) Retrieve the m_Array child.
+    std_array_elements = valobj.GetChildMemberWithName("m_Array").GetChildMemberWithName("__elems_")
```

Comment:
This uses the libc++ implementation which means it won't work with libstdc++ which is typical when using clang and lldb on linux.

Reply:
What is the other option?